### PR TITLE
Billboards endpoints Finished

### PR DESCRIPTION
**Pull request by Makinyel**
Billboards endpoints Finished

### CODE REVIEW

File: src/main/java/com/sellfinity/core/infrastructure/adapter/billboard/UpdateBillboardAdapter.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+@AllArgsConstructor
+public class UpdateBillboardAdapter implements UpdateBillBoardService {
+
+  private final BillboardRepository billboardRepository;
+  private final BillboardRepositoryMapper billboardRepositoryMapper;
+
+  @Override
+  public void updateBillBoard(Long id, Billboard billBoard) {
+    BillboardDto billboardDto = billboardRepositoryMapper.toDto(billBoard);
+    billboardRepository.save(billboardDto); //TODO USAR QUERY
```
Comment (waldo2810) [inline src/main/java/com/sellfinity/core/infrastructure/adapter/billboard/UpdateBillboardAdapter.java]: haz la query

File: src/main/java/com/sellfinity/core/infrastructure/api/billboard/BillboardRequestMapper.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import com.sellfinity.core.domain.entity.Billboard;
+import com.sellfinity.core.infrastructure.api.category.CategoryRequestMapper;
+import com.sellfinity.core.infrastructure.api.store.StoreRequestMapper;
+import com.sellfinity.core.shared.mapper.EntityToDto;
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingConstants.ComponentModel;
+
+@Mapper(componentModel = ComponentModel.SPRING, uses = {
```
Comment (waldo2810) [inline src/main/java/com/sellfinity/core/infrastructure/api/billboard/BillboardRequestMapper.java]: no necesitas ConponentModel

**Review by waldo2810 — CHANGES_REQUESTED** [review]

File: src/main/java/com/sellfinity/core/infrastructure/adapter/billboard/UpdateBillboardAdapter.java (hunk 1 above)
Comment (Makinyel) [reply src/main/java/com/sellfinity/core/infrastructure/adapter/billboard/UpdateBillboardAdapter.java]: Cule Flojo

File: src/main/java/com/sellfinity/core/infrastructure/api/billboard/BillboardRequestMapper.java (hunk 2 above)
Comment (Makinyel) [reply src/main/java/com/sellfinity/core/infrastructure/api/billboard/BillboardRequestMapper.java]: Que si joa

**Review by waldo2810 — APPROVED** [review]

**Pull request merged**